_posts/2020-05-04-introducing-open-sgid.md

```diff
@@ -12,32 +12,32 @@ tags:
+The timing could not be more perfect with the sudden increase in teleworking, which is not expected to wane any time soon, especially here in Utah with the implementation of the [Governor’s teleworking initiative](https://governor.utah.gov/2019/07/16/state-of-utah-introduces-teleworking-program-for-employees/). Not only does the Open SGID benefit Utahns, but it also signals the return to a truly publicly available, direct database connection that can be utilized on a multitude of GIS platforms. Furthermore, it initiates the sunsetting of the state network-only SGID10 database, although both databases (i.e., the Open SGID and the SGID10) will run in parallel for some time to allow users to transition and adapt to the Open SGID.
 
 The new tools and processes developed collectively by AGRC staff to accomplish the Open SGID, as well as the new and improved [Open Data site](https://opendata.gis.utah.gov/), are numerous and worthy of their own blogpost - perhaps some foreshadowing there. However, it is worth noting that a paradigm shift led to the creation of a new internal SGID database that serves as the sole editing environment and data source for the Open SGID and the Open Data site. Automated processes provide nightly updates to both data resources to maintain consistency between them, so you can be sure that you are utilizing the most current data available.
 
-A nearly month-long beta testing period was helpful to identify several issues and some inherent differences between the Open SGID and the enterprise geodatabase of the SGID10 era. A [GitHub repository](https://github.com/agrc/open-sgid) was created as the primary source of information for the Open SGID, and to catalog and address [these issues and differences](https://github.com/agrc/open-sgid/issues) with explanations or troubleshooting techniques.
+A nearly month-long beta testing period was helpful to identify several issues and some inherent differences between the Open SGID and the enterprise geodatabase of the SGID10 era. A [GitHub repository](https://github.com/agrc/open-sgid) was created as the primary source of information for the Open SGID and to catalog and address [these issues and differences](https://github.com/agrc/open-sgid/issues) with explanations or troubleshooting techniques.
 
-#### Known issues
+#### Known Issues
 
 - [Tables do not contain metadata](https://github.com/agrc/open-sgid/issues/5)
```

Comment:
If you wanted to leave these as complete sentences, I'd recommend adding periods after each.